clang/include/clang/Basic/TokenKinds.def

```diff
@@ -567,6 +567,7 @@ TYPE_TRAIT_1(__is_unsigned, IsUnsigned, KEYCXX)
 // Embarcadero Binary Type Traits
 TYPE_TRAIT_2(__is_same, IsSame, KEYCXX)
 TYPE_TRAIT_2(__is_convertible, IsConvertible, KEYCXX)
+TYPE_TRAIT_2(__is_nothrow_convertible, IsNothrowConvertible, KEYCXX)
```

Comment:
It would be nice to move __is_nothrow_convertible / __is_convertible / __is_same above the Embarcadero comment

Reply:
There are a bunch more type traits that aren't listed correctly anymore, so IMO an NFC patch that updates them all would be better.